enhancements/update/phased-rollouts.md

```diff
@@ -0,0 +1,316 @@
+The proposal also discusses the lack of Update Service pod state sharing, and the desire for a consistent response regardless of which Update Service pod ends up handling the request.
+Graph-admins who set the start time slightly in the future can give all Update Service pods sufficient time to ingest the new configuration before the window begins, to ensure the consistent responses.
+
+However, delaying the fast promotion by pushing back the start time introduces some divergence between the errata going public and the fast promotion, both of which are declarations of Red Hat support for the given release.
+There is some concern that users who have selected [the fast channel][fast-semantics] to get:
+
+> ... new ... versions as soon as Red Hat declares the given version as a general availability release.
+
+will be confused by even hour-scale delays between the errata going public and the appearance of update recommendations in their fast-channel clusters.
+
+We could instead promote to fast immediately upon receiving graph-data changes.
+If you assume that graph-data changes propagate to Update Service pods quickly compared to cluster-version operator polling, the chance of divergent graph reponses is small.
+The [current][cluster-version-operator-polling] cluster-version operator update polling period is between two and four minutes.
+And the [current][graph-builder-polling] Update Service graph builders pause for five minutes between scrapes with a five minute timeout on each scrape.
+That means that the cluster-version operator is likely to poll once or twice while the graph builder is paused between scrapes, even if the scrape itself is instantaneous.
+Sticky routing will keep a single cluster pinned to a single Update Service pod, but it will not mitigate situations where the previous Update Service pod has been removed (for example, because a configuration change is being rolled out).
+And of course it's possible that a stuck pod (broken node routing?  Dying graph-data image host?) could delay some Update Service pods from retrieving graph-data for even longer.
+
+In the rare cases when the cluster-version operator receives inconsistent graph responses as a graph-data change propagates, the impact should be small.
+Clusters with [automatic updates][automatic-updates] enabled should initiate an update before the next upstream poll.
+Clusters without automatic updates may have [the `UpdateAvailable` alert][cluster-version-operator-update-available] fire, resolve, and fire again.
+
+As another alternative, we could set start times slightly in the past, while keeping the fast promotion pinned to the configured start time.
+That would minimize the divergence between the errata going public and the appearance of update recommendations in their fast-channel clusters.
+But it would still introduce the same Update Service pod consistency issues.
```

Comment:
Also if we define the start time in the past that would create some inconsistency around the time window we provide for phased rollout.